addons/GME/Language/GME_localization.uk_ua.conf

```diff
@@ -21,7 +21,7 @@ StringTableRuntime {
   "GME-Editor_Attribute_Stance_Prone_Name"
   "GME-Editor_Attribute_Stance_Stand_Name"
   "GME-Editor_Attribute_ValueFormating_Length"
```

Comment:
Typo?

I must have missed it in the first review, as I couldn't see the change in the string-table source when looking through this review. (Which is why I flagged it in one of the generated files instead)

Reply:
Yeah, I missed it too...